manifest.yml

```diff
@@ -2,10 +2,8 @@
 applications:
 - name: ccs-scale-cat-service
```

Comment:
does the name need the ((route-prefix)) also?

Reply:
> does the name need the ((route-prefix)) also?

From: https://docs.cloudfoundry.org/devguide/deploy-apps/deploy-app.html#default-push

> The app name must be unique to your CF installation.
> - To use an app name that is not unique, customize the app’s route as described in Customize the Route below.
> - Apps running at their default routes require unique names because default routes are based on app names, and all routes must be globally unique.

Seems to imply the app name need not be unique so long as the route is - but happy to prefix it too for consistency..?

Reply:
No - that's fine - was just curious. Can see one app in the console, with 2 associated routes - does that mean the deployed app is shared by the routes? (i.e. if you update the app in sandbox, it will also be updated in dev)? Just wondering/curious

Reply:
The route without the prefix is the old one from before this change - I've been trying to remove it using the `cf delete-route` command but for some reason it thinks it doesn't exist.  Maybe simplest just to delete the app and redeploy..

Reply:
Ahh - sorry, getting my brain in a twist I think. I was thinking that was the dev route (not the old route) - didn't look properly - but of course, it isn't, and even so - develop will be viewed in an entirely different space/console (of course). Ignore me 🤦